src/libsystemd/sd-netlink/netlink-message.c

```diff
@@ -1093,38 +1093,39 @@ int sd_netlink_message_read_strv(sd_netlink_message *m, unsigned short container
         return 0;
 }
 
-static int netlink_container_parse(sd_netlink_message *m,
-                                   struct netlink_container *container,
-                                   struct rtattr *rta,
-                                   size_t rt_len) {
+static int netlink_container_parse(
+                sd_netlink_message *m,
+                struct netlink_container *container,
+                struct rtattr *rta,
```

Comment:
shouldn't this be const?

Reply:
Unfortunately, `RTA_NEXT()` macro drops `const` qualifier.